rpc/common/moc_common_common.proto

```diff
@@ -15,6 +15,8 @@ enum Operation {
 	IMPORT = 4;
 	EXPORT = 5;
 	VALIDATE = 6;
+	HYDRATE = 7;
```

Comment:
Lets not add properties that are not required.

Reply:
Since we are adding a separate operation for `HYDRATE`, this enum is required and used in `wssdagent` and `wssdcloudagent` repositories.